pkg/deploy/elbv2/load_balancer_manager.go

```diff
@@ -283,13 +297,25 @@ func buildSDKSecurityGroups(modelSecurityGroups []coremodel.StringToken) ([]*str
 	return sdkSecurityGroups, nil
 }
 
-func buildSDKSubnetMapping(modelSubnetMapping elbv2model.SubnetMapping) *elbv2sdk.SubnetMapping {
-	return &elbv2sdk.SubnetMapping{
-		AllocationId:       modelSubnetMapping.AllocationID,
+func buildSDKSubnetMapping(modelSubnetMapping elbv2model.SubnetMapping) (*elbv2sdk.SubnetMapping, error) {
+	ctx := context.Background()
```

Comment:
re-reading this code, wondering if ctx should get passed in since updateSDKLoadBalancerWithSubnetMappings has a context that we're not using here.